server/datastore/sql/migrations/002-add-comment-likes.sql

```diff
@@ -0,0 +1,16 @@
+CREATE Table comments (
+  id      VARCHAR PRIMARY KEY,
+  userId  VARCHAR NOT NULL,
+  postId  VARCHAR NOT NULL,
+  comment VARCHAR NOT NULL,
+  postedAt INTEGER NOT NULL,
+  FOREIGN KEY (userId) REFERENCES users (id),
+  FOREIGN KEY (PostId) REFERENCES posts (id)
+)
+
+CREATE Table likes (
+  userId  VARCHAR NOT NULL,
+  postId  VARCHAR NOT NULL,
+  FOREIGN KEY (userId) REFERENCES users (id),
+  FOREIGN KEY (postId) REFERENCES posts (id)
```

Comment:
we can also add a composite primary key of both fields, to let the database reject duplicate inserts. it's always a good idea to use a relational database's features rather than rely on application code to do it.